llvm/lib/Target/RISCV/RISCVInstrInfo.h

```diff
@@ -307,6 +307,13 @@ class RISCVInstrInfo : public RISCVGenInstrInfo {
   static bool isLdStSafeToPair(const MachineInstr &LdSt,
                                const TargetRegisterInfo *TRI);
 
+  static bool evaluateCondBranch(unsigned CC, int64_t C0, int64_t C1);
+
+  static bool isLoadImm(const MachineInstr *MI, int64_t &Imm);
```

Comment:
Do we need to define this function or can this be a lambda inside of `isFromLoadImm`? I think isFromLoadImm is the only caller of isLoadImm.

Reply:
yeah... `optimizeCondBranch` uses `isLoadImm` directly

Reply:
Ahh, got it.

Reply:
Should it be private function?

Reply:
yep, changing it now